bin/scraper-infra.ts

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env node
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import { SharedResourcesStack } from '../lib/shared-resources';
+import { DatabaseStack } from '../lib/database-stack';
+
+const app = new cdk.App();
+
+const dbStack = new DatabaseStack(app, 'DatabaseStack');
+
+const sharedResources = new SharedResourcesStack(app, 'ScraperInfraStack', {
+  /* If you don't specify 'env', this stack will be environment-agnostic.
+   * Account/Region-dependent features and context lookups will not work,
+   * but a single synthesized template can be deployed anywhere. */
+
+  /* Uncomment the next line to specialize this stack for the AWS Account
+   * and Region that are implied by the current CLI configuration. */
```

Comment:
We uncommented the next line already

```suggestion
  /* Specialize this stack for the AWS Account
   * and Region that are implied by the current CLI configuration. */
```